src/Uno.Extensions.Core.UI/Settings.cs

```diff
@@ -82,4 +82,31 @@ public void Set(string key, string value)
 #endif
 		}
 	}
+
+	public void Remove(string key) => Set(key, null);
+
+	public void Clear()
+	{
+		Initialize();
+		if (!_useFileSettings)
+		{
+			ApplicationData.Current.LocalSettings.Values.Clear();
+		}
+		else
+		{
+			settings.Clear();
+#if __WINDOWS__
+			File.WriteAllText(SettingsFile, JsonSerializer.Serialize(settings));
+#endif
+		}
+	}
+
+	public IReadOnlyCollection<string> Keys
+	{
+		get
+		{
+			Initialize();
+			return _useFileSettings ? settings.Keys : ApplicationData.Current.LocalSettings.Values.Keys.Select(k => k.ToString()).ToArray();
```

Comment:
What is the type of a key here?   (elsewhere it seems to already be strings)
```suggestion
			return _useFileSettings ? settings.Keys : ApplicationData.Current.LocalSettings.Values.Keys.Select(k => k.ToString(CultureInfo.InvariableCulture)).ToArray();
```